pkg/process/tagger/tagger.go

```diff
@@ -16,6 +16,7 @@ import (
 
 type TagResolver interface {
 	ResolveTags(appVersion *runtime.AuroraVersion, pushExtratags config.PushExtraTags) ([]string, error)
+	GetTags(appVersion *runtime.AuroraVersion, pushExtratags config.PushExtraTags) ([]string, error)
```

Comment:
Hva er forskjellen på de to metodene? dokumentasjon?

Reply:
Endret til ResoveShortTags. Den gir ut en slice med tags, f.eks (latest, 1, 1.0, 1.0.0)